service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43aSpecialGuardianshipOrderDocumentParameterGenerator.java

```diff
@@ -67,7 +69,7 @@ public DocmosisParameters generate(CaseData caseData) {
             .dateOfIssue(formatLocalDateTimeBaseUsingFormat(eventData.getManageOrdersApprovalDateTime(), DATE_TIME))
             .furtherDirections(eventData.getManageOrdersFurtherDirections())
             .orderDetails(getSpecialGuardianAppointeeMessage(caseData, selectedChildren.size()))
-            .orderByConsent(getOrderByConsentMessage(eventData))
+            .orderByConsent(orderMessageGenerator.getOrderByConsentMessage(eventData))
```

Comment:
B-e-a-u-tiful! @SooryaSu 
Great use of dependency injection and a great example of you demonstrating SOLID principles